meetup/models.py

```diff
@@ -1,3 +1,62 @@
+from uuid import uuid4
+
+from django.contrib.auth.models import User
 from django.db import models
 
-# Create your models here.
+
+class Venue(models.Model):
+    """This model is for the venue where be held"""
+    name = models.CharField(max_length=50)
+    location = models.CharField(max_length=255, null=True)
+    description = models.TextField(null=True)
+    latitude = models.FloatField()
+    longitude = models.FloatField()
```

Comment:
Use `PointField` or `DecimalField` instead of `FloatField`.
`FloatField` has not enough precision to use lat, lng.